src/main/kotlin/hu/kirdev/schpincer/web/ConfigureController.kt

```diff
@@ -408,6 +408,8 @@ class ConfigureController {
     @Throws(FileNotFoundException::class, DocumentException::class)
     fun showOpenings(openingId: Long,
                      type: String?,
+                     page: String?,
```

Comment:
I'd rename this variable to `pageOrientation` to reflect better what it stands for.